hooks/ruby/dismiss-review-server/server.rb

```diff
@@ -0,0 +1,115 @@
+require 'sinatra'
+require 'json'
+require 'uri'
+require 'net/http'
+
+$github_api_token = ENV['GITHUB_API_TOKEN']
+$github_secret_token = ENV['SECRET_TOKEN']
+
+post '/payload' do
+
+  # Only validate secret token if set 
+  if !$github_secret_token.nil?
+    payload_body = request.body.read
+    verify_signature(payload_body)
+  end
+
+  github_event = request.env['HTTP_X_GITHUB_EVENT']
+  if github_event == "push"
+    request.body.rewind
+    parsed = JSON.parse(request.body.read)
+
+    # Get branch information
+    branch_head = parsed['ref']
+    branch_name = branch_head.chomp("refs/heads")
```

Comment:
Great question. In that case, `branch_name` wouldn't result in any results when searching for Pull Requests and the code would happily continue on. Would you rather special case the `branch_head` and return?